### ISSUE TITLE
Failed to load the jupyterlab-git server extension

### ISSUE THREAD

**Issue by psychemedia** [jupyterlab/jupyterlab-git#1333]
Trying to run JupyterLab with the latest package in a clean Docker build, I get:

```text
Failed to load the jupyterlab-git server extension
The versions of the JupyterLab Git server frontend and backend do not match. The @jupyterlab/git frontend extension has version: 0.50.0 while the python package has version 0.50.1. Please install identical version of jupyterlab-git Python package and the @jupyterlab/git extension. Try running: pip install --upgrade jupyterlab-git
```

<img width="483" alt="image" src="https://github.com/jupyterlab/jupyterlab-git/assets/82988/c11ed521-7760-40ec-9bd9-af21ca87942c">

**Comment by jab** [comment]
Just tried installing this extension for the first time, in a freshly-installed JupyterHub instance, and immediately hit this issue. Are you able to reproduce this @zsailer et al.?

**Comment by akashkroy** [comment]
I had the same issue in jupyterhub. Restarted the server resolved it.

**Comment by jab** [comment]
Restarting did not resolve the issue for me.

**Comment by fcollonval** [comment]
i don't know if this was the reason. But I saw that version 0.50.1 failed to be published on conda-forge. It is done now. Does that solve your issue?

**Comment by jab** [comment]
No, that didn’t fix this issue. It should be easy to check that this is still broken. Just try installing jupyterlab-git in a fresh jupyterlab instance (eg using the Extensions tab in the UI) then observe that every time you load the UI you get the error “Failed to load the jupyterlab-git server extension”.

**Comment by krassowski** [comment]
@jab I deployed hundreds of JupyterLab servers and did not encounter this issue. Your problem may be that you are installing the extension too late, this is after the single user server has started. Alternatively, you may be running the legacy notebook server instead of jupyter-server. Running `jupyter server extension list`, pasting startup logs from the single user server and the JupyterHub configuration may help to give us better picture here.
